configurations/pipeline_config.json

```
@@ -0,0 +1,156 @@
+{
+  "PipelineName": "KE-Constitution-Review",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/ke-vax-textit-token.txt",
+      "ActivationFlowNames": [
+        "ke_constitution_review_poll",
+        "ke_constitution_review_poll_catch_all"
+      ],
+      "SurveyFlowNames": [
+
+      ],
+      "TestContactUUIDs": [
+        "74efc650-b89d-4d66-89c5-0e9a99912b8a",
+        "7aaf3325-6222-4d9e-b7ed-93b4ef78e5be",
+        "93b43c8c-e261-4cff-b044-1bc5e6c3d5a1",
+        "46bc7104-9a23-42d2-8b04-f2570990b518",
+        "ee71626e-9b39-439e-be2f-1a45441209ed",
+        "ca8ab486-22f1-4004-a192-3e4ae8220a00"
+      ]
+    },
+    {
+      "SourceType": "GCloudBucket",
+      "ActivationFlowURLs": [],
+      "SurveyFlowURLs": [
+        "gs://avf-project-datasets/2021/Kenya-Pool/covid19_ke_urban_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/covid19_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/oxfam_wash_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/undp_kenya_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/unicef_covid19_ke_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/worldvision_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/gpsdd_bungoma_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/gpsdd_kilifi_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/gpsdd_kiambu_s01_demog.jsonl"
+      ]
+    }
+  ],
+  "ListeningGroupCSVURLs": [],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "KE-VAX_phone_number_avf_phone_id"
+  },
+  "OperationsDashboard": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json"
+  },
+  "RapidProKeyRemappings": [
+    {"RapidProKey": "avf_phone_id", "PipelineKey": "uid"},
+
+    {"RapidProKey": "appointment_views (Text) - ke_constitution_review_poll", "PipelineKey": "appointment_views_raw", "IsActivationMessage": true},
```

```suggestion
    {"RapidProKey": "Appointment_Views (Text) - ke_constitution_review_poll", "PipelineKey": "appointment_views_raw", "IsActivationMessage": true},
```
The keys are usually Snake cased even though the variable names are not